tests/stats.test.ts

```diff
@@ -0,0 +1,180 @@
+import { buildApiServer } from '../src/api/init';
+import { cycleMigrations } from '../src/pg/migrations';
+import { PgStore } from '../src/pg/pg-store';
+import { DbInscriptionCountPerBlock } from '../src/pg/types';
+import { TestChainhookPayloadBuilder, TestFastifyServer } from './helpers';
+
+jest.setTimeout(100_000_000);
```

Comment:
Was this done for testing purposes? We should remove this to avoid infinite wait times on Github actions 😄

Reply:
Yep, I forgot to remove this -- thx 🙏